src/scripts/region-redirects.js

```diff
@@ -3,14 +3,28 @@ import config from '../../regions.config';
 
 // need to wait for DOM since this script is loaded in the <head>
 document.addEventListener('DOMContentLoaded', () => {
-    const regionSelector = document.querySelector('.js-region-selector');
+    const regionSelector = document.querySelector('.js-region-select');
 
     if (regionSelector) {
-        regionSelector.addEventListener('change', regionOnChangeHandler);
+        const options = regionSelector.querySelectorAll('.dropdown-item');
+
+        options.forEach(option => {
+            option.addEventListener('click', () => {
+                const region = option.dataset.value;
+                regionOnChangeHandler(region)
+            })
+        })
     }
 });
 
-function isReferrerEU(){
+function replaceButtonInnerText(value) {
+    const selectedRegion = value === 'gov' ? 'US1-FED' : value.toUpperCase();
```

Comment:
can we set `selectedRegion` here from the config value dd_datacenter?

Reply:
@nsollecito this is much better.  thanks!